### DISCUSSION TITLE
Help for Prowlarr Setup

### DISCUSSION THREAD

**Discussion by devasenan134** [mhdzumair/MediaFusion#234] [Q&A]
I'm was trying to get some indexers for tamil movies on prowlarr, with which i ended up finding this repo!
But i can't figure out about setting up this media fusion as custom indexer on prowlarr 
Can someone please, guide me through the steps

**Comment by mhdzumair** [comment]
Partial implementation available here https://github.com/mhdzumair/MediaFusion/pull/218
